tw.ddnet.ddnet.yml

```diff
@@ -0,0 +1,58 @@
+id: tw.ddnet.ddnet
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
```

Comment:
Your manifest should define `command` as well.

Reply:
Fixed. Thanks :+1: